Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+pipeline {
+    agent label:"docker", docker: "maven:3.3.9-jdk-7"
+
+    environment {
+      MAVEN_OPTS = "-XX:MaxPermSize=256m -Xmx1024m"
+    }
+
+    stages {
+        stage("Build") {
+            sh 'mvn clean install -Dmaven.test.failure.ignore=true'
+        }
+    }
+
+    postBuild {
+        always {
```

Comment:
Even if the build fails? Will this not pick up artifacts from an older build using the same workspace?